docs/start/getting-started/fragments/android/setup.md

```diff
@@ -70,11 +71,12 @@ Rather than configuring each service through a constructor or constants file, th
 Add a [GraphQL API](https://docs.aws.amazon.com/appsync/latest/devguide/designing-a-graphql-api.html) to your app and automatically provision a database by running the the following command from the root of your application directory:
 
 ```bash
-amplify add api #accept defaults
+amplify add api
 ```
 
-The default values are highlighted below.
-```bash
+Accept the default values which are highlighted below:
```

Comment:
user studies have shown that people don't read much outside of codeblocks. we should make these either bold or retain them in comments

Reply:
I'll bold **default values** but this concern seems off base to me as there's a code block below.